src/test/java/seedu/address/model/group/MeetingTest.java

```diff
@@ -0,0 +1,71 @@
+package seedu.address.model.group;
+
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
+
+import static seedu.address.testutil.TypicalMeetings.URGENT;
+import static seedu.address.testutil.TypicalMeetings.WEEKLY;
+
+import org.junit.Test;
+
+import seedu.address.testutil.MeetingBuilder;
+
+
+public class MeetingTest {
+
+    @Test
+    public void isSameMeeting() {
```

Comment:
another test case could be another object with the same variables and check if the objects are of equivalent.

Reply:
Ok, noted. Thanks!